dev_guide/idling_unidling_services.adoc

```diff
@@ -0,0 +1,63 @@
+[[dev-guide-idling-unidling-services]]
+= Idling and Unidling Services
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+You can idle services in order to reduce the consumption of resources.
```

Comment:
I'd say "you can idle applications fronted by services" (since the `oc idle` command takes in service names, but the idling action involves the whole "application" (service + dc/rc))